### PR TITLE
Adding Event URL in manage event section

### LINKED ISSUE #2632: External Event URL is not shown in Manage Events
**Describe the bug**

When a event has external ticket URL, Its details are not shown in `Manage Events` Section

**To Reproduce**

Steps to reproduce the behaviour:

1. Go to 'Admin/Events/All'
2. See error

**Expected behaviour**

Add `External Event URL` under this column

**Screenshots**

image

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Chrome
 - Version: 73.0.3

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Solves #2632 

#### Changes proposed in this pull request:

- Added an `if clause` to check for event url

Fixes #2632

### CODE REVIEW

**Comment by kushthedude** [comment]
@ritikamotwani @CosmicCoder96 Review

**Comment by mohitm15** [comment]
What sort of details you are talking about in the issue

**Comment by kushthedude** [comment]
See the issue

**Comment by kushthedude** [comment]
@CosmicCoder96 Ready for review

**Comment by kushthedude** [comment]
@CosmicCoder96 @ritikamotwani Review Please

File: app/templates/components/ui-table/cell/cell-tickets.hbs [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 {{else}}
-  <div>{{t 'No Ticket Information'}}</div>
+  {{#if record.tickets}}
+    <div class="ui list">
+      {{#each record.tickets as |ticket|}}
+        <div class="item">{{ticket.name}} ({{ticket.orderStatistics.tickets.completed}}/{{ticket.quantity}}) <span class="item muted text">[{{ticket.type}}]</span></div>
+      {{/each}}
+    </div>
+  {{else}}
+    <div>{{t 'No Ticket Information'}}</div>
```
Comment (CosmicCoder96) [inline app/templates/components/ui-table/cell/cell-tickets.hbs]: no need for the `div` tag.

File: app/templates/components/ui-table/cell/cell-tickets.hbs [hunk 2]
```diff
@@ -1,9 +1,13 @@
-{{#if record.tickets}}
-  <div class="ui list">
-    {{#each record.tickets as |ticket|}}
-      <div class="item">{{ticket.name}} ({{ticket.orderStatistics.tickets.completed}}/{{ticket.quantity}}) <span class="item muted text">[{{ticket.type}}]</span></div>
-    {{/each}}
-  </div>
+{{#if record.ticketUrl}}
+  <div>{{t 'You have to go to '}}<a href="{{record.ticketUrl}}">{{record.ticketUrl}}</a>{{t ' to check ticket info'}}</div>
```
Comment (CosmicCoder96) [inline app/templates/components/ui-table/cell/cell-tickets.hbs]: no need for the `div` tag.

Comment (CosmicCoder96) [inline app/templates/components/ui-table/cell/cell-tickets.hbs]: Change the text to `Vist \<link\> to book tickets.`

Comment (kushthedude) [reply app/templates/components/ui-table/cell/cell-tickets.hbs]: Will make certain changes.

**Comment by kushthedude** [comment]
@CosmicCoder96 did the changes,Review Pleae

**Pull request merged**